executable/executable_test.go

```diff
@@ -1,16 +1,29 @@
 package executable
 
 import (
-	"github.com/stretchr/testify/assert"
+	"os"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestStart(t *testing.T) {
 	err := NewExecutable("/blah").Start()
-	assertErrorContains(t, err, "no such file")
+	assertErrorContains(t, err, "not found")
 	assertErrorContains(t, err, "/blah")
 
+	// Permissions are not preserved across remote git repos.
```

Comment:
@ryan-gang what does this mean? If you just had a file that isn't executable it should remain non-executable in a git repo, no?